config/staging.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  VerifierRegistry: {
+    tokenAddress: '0x1588d300a9995934aa9daae19be285a66eb46c1c',
+    verifiersPerShard: 5,
+  },
+  ContractRegistryAddress: '0x3c5c8a1a2ef948e97495b785eb4b45c9de70911e',
```

Comment:
I think something like this is cleaner and avoid polluting the top-level config object.

```
ContractRegistry: {
  address: '0x3c5c8a1a2ef948e97495b785eb4b45c9de70911e',
  storage: '0x4e7f37c962bf4fb8aa1c7de57a6b33dd0a6a2e53'
}
```